docs/source/extension/extension_points.rst

```diff
@@ -457,16 +457,111 @@ The recommended ranges for this rank are:
 Main Menu
 ---------
 
-There are three main ways to extend JupyterLab's main menu.
+There are two ways to extend JupyterLab's main menu.
```

Comment:
As a follow-up, it would be nice to also have an example in https://github.com/jupyterlab/extension-examples.

Or we could also update the existing one about menus: https://github.com/jupyterlab/extension-examples/tree/master/main-menu